### PR TITLE
Feature/examples

### PR DESCRIPTION
**Pull request by DAtink**

### CODE REVIEW

File: Examples/4-orm/Queries/GreetingsValuesQuery.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    public GreetingsValuesQuery(int? id)
+    {
+      var resultSql = sql;
+
+      if (id.HasValue)
+      {
+        resultSql += condition;
+
+        Parameters = new Dictionary<string, object>
```
Comment (PavelKulbida) [inline Examples/4-orm/Queries/GreetingsValuesQuery.cs:19]: а точно object а не int?

File: Examples/4-orm/GreetingsController.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrmService.Entities;
+using ViennaNET.Orm.Application;
+using ViennaNET.Utils;
+
+namespace EmptyService
```
Comment (sergeyo) [inline Examples/4-orm/GreetingsController.cs:7]: Empty service
you may just use reshaper/vs command which will fix namespace issues in whole project

**Comment by RomanProsin** [comment]
Отличная работа.

File: Examples/4-orm/GreetingsController.cs (hunk 2 above)
Comment (DAtink) [reply Examples/4-orm/GreetingsController.cs]: Fixed

**Pull request merged**